modules/social_features/social_group/modules/social_group_request/social_group_request.install

```diff
@@ -40,11 +40,11 @@ function social_group_request_update_dependencies(): array {
  */
 function _social_group_request_set_permissions(): void {
   if (\Drupal::moduleHandler()->moduleExists('social_group_flexible_group')) {
-    /** @var \Drupal\group\Entity\GroupRoleInterface $outsider */
-    $outsider = \Drupal::entityTypeManager()
+    /** @var \Drupal\group\Entity\GroupRoleInterface $verified */
```

Comment:
@rochek03 you mentioned `social_group_request_update_13003` in the PR description.
Was it removed in the end? If so - please update PR description or add it back if needed ;)

Reply:
Fixed